variables/_svgs.pcss

```diff
@@ -247,6 +304,24 @@
 	}
 }
 
+@svg-load search-focus url('../icons/search.svg') {
+
+	path {
+		fill: var(--color-icon-focus);
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * SVG - Search Filter
+ * ----------------------------------------------------------------------------- */
+
+@svg-load search-filter url('../icons/search-filter.svg') {
+
+	path {
```

Comment:
The svg has a `<circle>` element, but we're not overriding the color. It also uses stroke rather than fill. After minifying svgs, make sure that these color overrides work properly (it may not have a `<circle>` element anymore).